include/solvers/multiphysics_interface.h

```diff
@@ -493,6 +493,52 @@ class MultiphysicsInterface
     return physics_solutions[physics_id];
   }
 
+  /**
+   * @brief Request the average solution of a given physics
+   *
+   * @param physics_id The physics of the solution being requested
+   */
+  TrilinosWrappers::MPI::Vector *
+  get_average_solution(PhysicsID physics_id)
+  {
+    AssertThrow((std::find(active_physics.begin(),
+                           active_physics.end(),
+                           physics_id) != active_physics.end()),
+                ExcInternalError());
+    return physics_average_solutions[physics_id];
+  }
+
+  /**
+   * @brief Request the present block average solution of a given physics
+   *
+   * @param physics_id The physics of the solution being requested
+   */
+  TrilinosWrappers::MPI::BlockVector *
+  get_block_average_solution(PhysicsID physics_id)
+  {
+    AssertThrow((std::find(active_physics.begin(),
+                           active_physics.end(),
+                           physics_id) != active_physics.end()),
+                ExcInternalError());
+    return block_physics_average_solutions[physics_id];
+  }
+
+
+  /**
+   * @brief Request the reynolds_stress solution of a given physics. This is WIP and is not yet implemented in the solver.
```

Comment:
```suggestion
   * @brief Request the reynolds_stress solution of a given physics. 
   * WIP for an upcoming PR, not yet implemented in the solver.
```

Reply:
not sure it should stay in this PR though, maybe you could start a new branch for the next PR right away and discard it in this PR?

Reply:
I guess the comment can remain here, it's not really exposed to the user anyway.